admin_ui/app/permission/PermissionForm.js

```diff
@@ -126,18 +113,15 @@ const PermissionForm = ({ onSubmit, onCancel, onDelete, type }) => {
           />
           {errors.org && <p className="text-red mt-2">{errors.org}</p>}
         </div>
-      </div>
-      <div className="my-4 flex">
-        <div className="w-1/4 flex-shrink-0">&nbsp;</div>
-        <div className="flex items-center">
+        <div className="form-submit">
```

Comment:
Nit-picky suggestion: consider `form-buttons` or `form-controls` instead of `form-submit`. Seems more correct from a technical standpoint.

Your call.